media-gfx/openvdb/files/openvdb-4.0.2-fix-build-docs.patch

```diff
@@ -0,0 +1,12 @@
+diff -Naur a/openvdb/CMakeLists.txt b/openvdb/CMakeLists.txt
```

Comment:
Could you add a wee bit of context to the top of these patches?

- Link to upstream bug if any (there should be one if you can, we prefer to upstream all patches)
- Link to a Gentoo bug

This [guidance](https://dev.gentoo.org/~vapier/clean-patches) is pretty neat.

Reply:
I have included more details at the top of the patch, roughly along the lines of my commit message. This patch is only needed for openvdb 4 and 5, where ADD_CUSTOM_COMMAND is used. In openvdb 6 and 7 the documentation was moved to a separate directory and added by add_subdirectory(doc) so it is not required there.